### DISCUSSION TITLE
dark high contrast theme index page Copilot part inaccessible

### DISCUSSION THREAD

**Discussion by spookyorange** [community/community#19202] [Accessibility] [answered]
![Screenshot from 2022-06-22 22-07-09](https://user-images.githubusercontent.com/82583304/175118277-740e53cc-1790-436d-93e8-fefe4a0a1318.png)

Fedora Workstation 35 // GNOME // X11 // Firefox 101.0.1 (64-bit)

Like in the attachment in dark high contrast theme Copilot paragraph is unreadable.

**Comment by benjiallen** [comment, marked as answer]
Thanks for letting us know.

**Comment by cehfisher** [comment]
Hi @spookyorange - it looks like this ad is no longer on the main github.com page. The current ads meet the color contrast ratio requirement for its text size. Thank you for letting us know about this issue in the past!

@queenofcorgis - this issue can be closed as resolved.
